src/schedule/2021-01-20-add-youtube-and-twitter-shortcodes-to-eleventy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Add YouTube and Twitter Shortcodes to Eleventy
+type: Mob Job
+date: 2021-01-20T10:00:00-08:00
+---
+
+[chantastic](https://twitter.com/home) and the Lunch Dev crew add YouTube and Twitter shortcodes to [events.lunch.dev](https://events.lunch.dev).
+
+The goal: instead of copying-pasting large embed codes into markdown files, new posts can use shortcodes like `{% youtube 'https://youtu.be/S0M31D' %}` and `{% twitter 'htttps;//twitter.com/status/4815162342 %}` and take advantage of responsive container styles.
```

Comment:
It looks like the build is trying to treat the `{% %}` here as actual Liquid tags, and it's failing the build since we don't have `youtube` or `twitter` filters yet.

Can we reword this or retool this so our builds don't fail?

Reply:
Ahh, here's the fix: wrapping this code in Liquid's [`{% raw %}`](https://shopify.github.io/liquid/tags/raw/) filter

Reply:
thanks for the tip!
i tried `raw` without much luck. possible i was just missing something obvious.
will definitely revisit when i attach the video… using the freshly minted shortcodes #metariffic!